pages/browse-our-faq/bitmark-protocol-faq.md

```diff
@@ -0,0 +1,59 @@
+
+To solve this problem, the provenance of data must be secured. That way information can be built and the origin and history can always be authenticated. That is the goal of the Bitmark Protocol. It has been in production use since 2018. Already, organizations like UC Berkeley, KKBOX, and Pfizer use the Bitmark Protocol to increase trust in valuable data. In total, the protocol is securing more than one million data records in healthcare, music, art, finance, and more.
+
+## Why secure data provenance?
+
+To alleviate the externalities of the Internet, we must have a system to control information. One option is to go “centralized” with government regulation and corporate Oversight Boards that are tasked with making new rules and then moderating content. However, this is a global issue, so State-level regulation will often conflict across different countries. The other option is to go “decentralized” and build an end-to-end protocol that secures the origin and history, or “provenance” of data itself. The benefits are:
+
+* When data has provenance, information (processed data) can be authenticated without a trusted third party. 
+* End-to-end protocols are low-cost.
+* Having a global data provenance enhances local regulation (more below).
+
+## Why hasn’t this been done before?
+
+End-to-end protocols cannot rely on trusted third parties. In order to have such a protocol for information ownership/control we must have the ability to secure the provenance of data. Information is processed data. Before Bitcoin (specifically public blockchains), there was not a way to secure the provenance of anything digital without relying upon trusted third parties. 
+
+## How can information be controlled?
+
+The Bitmark protocol provides a set of rules governing how data can be held, used, and exchanged. This works because the data itself can be held locally or in the Cloud. The protocol provides safe, low-cost access control without requiring a trusted third party.
+
+When changes or ownership / control are needed, the underlying data can be transferred peer-to-peer, creating an unforgeable provenance. 
+
+## How can this protect digital rights? 
+
+Information is the key to human rights. And the key to controlling information is property rights. 
+
+Property rights are agreements of control across different trust boundaries. For example, you could have a property right to access a river running through your land. In the digital environment, a tech company could recognize a property right to use your data in their algorithm. 
+
+For most of human history, the dominant form of property has been personal property. The functionality of personal property has not usually depended on trusted third parties. Today, an increasing share of our personal property is digital, and our rights depend on BigTech. An end-to-end protocol for information control could record digital property rights and provide a better future. 
+
```

Comment:
I'd suggest against using casual phrases like "BigTech" in docs. Perhaps "large, centralized tech firms" or something similar.